modules/create_timepoint/php/create_timepoint.class.inc

```diff
@@ -205,7 +205,7 @@ class Create_Timepoint extends \NDB_Form
 
                 $languages = \Utility::getLanguageList();
                 if (count($languages) > 1) {
-                    array_unshift($languages, null);
+                    $languages = [null] + $languages;
```

Comment:
@laemtl I'm a little confused since I've never seen it done this way before. So I did a quick test in php with the following:

```
<?php
// Your code here!

$test =['a','b','c'];
print_r($test);

array_unshift($test, null);
print_r($test);

$test =['a','b','c'];

$test = [null] + $test;
print_r($test);

?>
```

The output:
```
Array
(
    [0] => a
    [1] => b
    [2] => c
)
Array
(
    [0] => 
    [1] => a
    [2] => b
    [3] => c
)
Array
(
    [0] => 
    [1] => b
    [2] => c
)
```

It looks like your way replaces the first element with null?

Reply:
Indexes start from 1 in Mysql, this case never occurs.

Reply:
@laemtl yes and no, a database admin could set the field to `0` it's a valid value. I think the solution is easy
```suggestion
                    $languages = [null=>null] + $languages;
```

I havent tried it but this should fix both of your issues